modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/settings/data-privacy/hocs/RequestList.tsx

```diff
@@ -263,7 +263,9 @@ const RequestList: React.FC<IRequestListProps> = ({
 						emailAddresses,
 						fileName,
 						ownerId: currentUser.id,
-						types
+						types,
+						userId: currentUser.userId,
+						userName: currentUser.screenName
```

Comment:
Hey @lesliewong92 can we use screenName from the logged user here?

Reply:
Get username from `currentUser.name` instead.

Reply:
Thanks for you review @interaminense, done!